packages/cli/generators/app/templates/public/index.html.ejs

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title><%= project.description %></title>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" type="image/x-icon" href="//v4.loopback.io/favicon.ico">
+
+  <style>
+    h3 {
+      margin-left: 25px;
+      text-align: center;
+    }
+
+    a, a:visited {
+      color: #3f5dff;
+    }
+
+    h3 a {
+      margin-left: 10px;
+    }
+
+    a:hover, a:focus, a:active {
+      color: #001956;
+    }
+
+    .power {
+      position: absolute;
+      bottom: 25px;
+      left: 50%;
+      transform: translateX(-50%);
+    }
+
+    .info {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%)
+    }
+
+    .info h1 {
+      text-align: center;
+      margin-bottom: 0px;
+    }
+
+    .info p {
+      text-align: center;
+      margin-bottom: 3em;
+      margin-top: 1em;
+    }
+  </style>
+</head>
+
+<body>
+  <div class="info">
+    <h1><%= project.name %></h1>
+    <p>Version <%= project.version || '1.0.0' %></p>
```

Comment:
This was intentional as the first step:

We wanted to keep the HomePageController implementation simple and easy to understand.
  - The generated `public/index.html` is a plain html instead of a template. We set the name/version during code gen but it's up to developers to maintain it
  - There is no need to use a template engine to render the view

That's been said, I'm fine to upgrade it to an EJS view once https://github.com/strongloop/loopback-next/pull/1764 is landed.

Reply:
Can we remove the Version from the static page then? Application name is unlikely to change, but the version is going to change often.